Liubou.Zviaruha/L2/index.css

```diff
@@ -0,0 +1,62 @@
+.sticky {
+    position: fixed;
+    top: 0;
+    width: 100%;
+}
+
+.h2 {
+    font-size: 200%;
+    font: 'Acme', sans-serif;
+    color: green;
+}
+
+.img0 {
+    min-height: 200px;
+    height: 300px;
+    width: 1360px;
+    padding: 10px;
+    margin-left: -70px;
+}
+
+.img1 {
+    float: left;
+    width: 250px;
+    height: 200px;
+    margin: 10px;
+}
+
+.img2 {
+    float: right;
+    width: 700px;
+    height: 500px;
+}
+
+.pre {
+    padding: 20px;
+    max-width: 600px;
+    background: #A9A9A9;
+    border: 4px double black;
+    font-family: 'IBM Plex Mono', monospace;
+}
+
+.section {
+    margin-top: 450px;
+    padding: 0 100px 0 100px;
+}
+
+.img3 {
+    width: 30px;
+    height: 30px;
+    float: right;
+}
+
+.body {
+    background-color: blanchedalmond;
+}
+.p {
```

Comment:
And here too, and in other places. More concrete :) What and in which place. Sort of like BEM suggests to do so.
http://getbem.com/naming/